Tests/LibCore/TestLibCoreIODevice.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/File.h>
+#include <LibTest/TestCase.h>
+#include <unistd.h>
+
+TEST_CASE(file_readline)
+{
+    auto path = "/res/txt/long-line.txt";
```

Comment:
This file is called `long-lines.txt`, not `long-line.txt`.

Reply:
Also, this seems to be written to be run only inside serenity, is it not being run with Lagom?

Reply:
> This file is called `long-lines.txt`, not `long-line.txt`.

Fixed

Reply:
> Also, this seems to be written to be run only inside serenity, is it not being run with Lagom?

Working on it, I'm getting:

/usr/bin/ld: Userland/Libraries/LibCore/libcore.so: error adding symbols: file in wrong format

Reply:
> Also, this seems to be written to be run only inside serenity, is it not being run with Lagom?

I think I managed to make it run with Lagom:

      Start  4: test-iodevice
 4/89 Test  #4: test-iodevice ........................   Passed    0.00 sec